README.md

```diff
@@ -57,36 +30,57 @@ gem 'xdrgen'
 
 And then execute:
 
-    $ bundle
+```
+$ bundle
+```
 
 Example usage:
 
 ```ruby
 require 'xdrgen'
```

Comment:
Telling folks to use the gem as published on xdrgen doesn't really work today because this library hasn't been released to rubygems since 2021 and the released version doesn't have features like the custom generator.

I'm working with ops to get this repo setup to be able to publish again:
- https://stellarfoundation.slack.com/archives/C02U19A2A/p1747716040439709?thread_ts=1747715993.367239&cid=C02U19A2A

Reply:
Tracking:
- https://github.com/stellar/xdrgen/issues/224